<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parthenos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>File selection</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>751</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="add_files">
           <property name="text">
            <string>add files</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="add_dir">
           <property name="text">
            <string>add directory</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="goto_archive">
           <property name="text">
            <string>Go to Archiving files</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <widget class="QPushButton" name="add_to_index">
           <property name="text">
            <string>add to index</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="treeView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <attribute name="title">
      <string>Archiving files</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>File upload</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>F&amp;ile</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose_pickling"/>
    <addaction name="actionSettings"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Commands</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Oper&amp;ations</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>&amp;Parameters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionFind_Action"/>
    <addaction name="actionHelp"/>
    <addaction name="actionGetting_started"/>
    <addaction name="actionKeymap_referance"/>
    <addaction name="actionRegistre"/>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOperations"/>
   <addaction name="menuParameters"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="separator"/>
   <addaction name="actionextract"/>
   <addaction name="separator"/>
   <addaction name="actionsend"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;New pickling...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Open pickling...</string>
   </property>
  </action>
  <action name="actionFind_Action">
   <property name="text">
    <string>&amp;Find Action</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionGetting_started">
   <property name="text">
    <string>&amp;Getting started</string>
   </property>
  </action>
  <action name="actionKeymap_referance">
   <property name="text">
    <string>&amp;Keymap reference</string>
   </property>
  </action>
  <action name="actionRegistre">
   <property name="text">
    <string>&amp;Registre</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>&amp;Check for updates</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>&amp;about</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="actionClose_pickling">
   <property name="text">
    <string>&amp;Close pickling</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Sett&amp;ings</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ADD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionextract">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/extract.png</normaloff>:/extract.png</iconset>
   </property>
   <property name="text">
    <string>extract</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionsend">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/send.png</normaloff>:/send.png</iconset>
   </property>
   <property name="text">
    <string>send</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Send&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
